### PR TITLE
Fixed variable heights of cards w/signoff

### LINKED ISSUE #1955: [ Learn-ng ] Fix container to be even
**Current Behavior**
As a result of the text "The mastering service meshes for Developer"  wrapping, the container appears longer than the other container.
Screenshot 2021-06-25 at 16 20 25
**Desired Behavior**
Adjust The "The mastering service meshes for Developer"  container to be even In length with the other containers.

---
**Screenshots / Mockups**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by parthlaw**
Signed-off-by: Parth <parthlaw24@gmail.com>

**Description**

This PR fixes #1955 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]
Screenshot 2021-06-28 at 7 15 35 PM

That definitely fixes the issue 🙌🏽 . 
Can you also makes those icons aligned eqaully ?

**Comment by leecalcote** [comment]
This is great to see. 👍

File: src/components/Learn-Components/Card-Component/learn-card.style.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const CardWrapper = styled.div`
 
     .card-parent{
         border-radius: 2rem;
+        min-height:357px;
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/Card-Component/learn-card.style.js:15]: Use rem instead.

File: src/components/Learn-Components/Card-Component/learn-card.style.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ const CardWrapper = styled.div`
...
+        display:flex;
+        flex-direction:column;
+        justify-content:space-between;
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/Card-Component/learn-card.style.js:18]: Is the flex-direction and justify-content really needed?

**Comment by l5io** [comment]
🚀 Preview for commit 388964480ea07dc5ad163853fdde37befc7e04a5 at: https://60db7dafa307c900bd154d84--layer5ng.netlify.app

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Do incorporate the above mentioned changes.

File: src/components/Learn-Components/Card-Component/learn-card.style.js (hunk 2 above)
Comment (parthlaw) [reply src/components/Learn-Components/Card-Component/learn-card.style.js:18]: Yes..They are to align the icons to the bottom so that they fall in line.

**Comment by l5io** [comment]
🚀 Preview for commit 886173019dc7df7f446e10069541618fc56a3e3f at: https://60deee6074fefbbd622007df--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**